### PR TITLE
Code comments, cleanup, nuget package info

### PR DESCRIPTION
**Pull request by tmarkovski**
Small maintenance update

### CODE REVIEW

File: wrappers/dotnet/Directory.Build.props [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    <PropertyGroup>
+        <Authors>Tomislav Markovski</Authors>
+        <Company>Hyperledger</Company>
+        <PackageLicenseExpression>Apache-2.0</PackageLicenseExpression>
+        <PackageProjectUrl>https://github.com/mattrglobal/ffi-bbs-signatures</PackageProjectUrl>
+        <PackageTags>BBS+ Ursa ZKP</PackageTags>
+        <Product>BBS+ Signatures</Product>
+        <RepositoryUrl>https://github.com/mattrglobal/ffi-bbs-signatures.git</RepositoryUrl>
+        <RepositoryType>git</RepositoryType>
+        <Version>1.0.0</Version>
```
Comment (tplooker) [inline wrappers/dotnet/Directory.Build.props:12]: How do feel about keeping the version pre-1.0 i.e if we decide to keep the wrappers minor version in lockstep with each other then the first released version would be 0.1.0?

File: wrappers/dotnet/src/Hyperledger.Ursa.BbsSignatures/Keys/BlsKeyPair.cs [hunk 2]
```diff
@@ -70,24 +77,44 @@ public BlsKeyPair(byte[] keyData)
...
+                return new BbsKey(context.ToByteArray(publicKey), messageCount);
             }
         }
+
+        /// <summary>
+        /// Creates new <see cref="BlsKeyPair"/> using a input seed as byte array.
+        /// </summary>
+        /// <param name="seed">The seed.</param>
+        /// <returns></returns>
+        public static BlsKeyPair Generate(string? seed = null)
```
Comment (tplooker) [inline wrappers/dotnet/src/Hyperledger.Ursa.BbsSignatures/Keys/BlsKeyPair.cs:105]: Just for context in this library there will soon be support for BLS12-381 G1 and G2 key gen support so if you want to account for that in this API now then take a look at the [objective-c](https://github.com/mattrglobal/ffi-bbs-signatures/pull/15) wrapper

**Review by tplooker — COMMENTED** [review]
Couple of minor comments, looks great though!

File: wrappers/dotnet/src/Hyperledger.Ursa.BbsSignatures/Keys/BlsKeyPair.cs (hunk 2 above)
Comment (tmarkovski) [reply wrappers/dotnet/src/Hyperledger.Ursa.BbsSignatures/Keys/BlsKeyPair.cs:105]: I was thinking about adding a field to denote the specific curve in the `BlsKeyPair` class, which should become `record` once net5 comes out. You think that's a good approach?

Comment (tplooker) [reply wrappers/dotnet/src/Hyperledger.Ursa.BbsSignatures/Keys/BlsKeyPair.cs:105]: Yeap I think both is worth doing

1. Add a member to the BLSKeyPair that denotes the field G1 or G2 of a generated key
2. Either add a parameter to the generate function to require what field you want to generate the key for or have seperate functions for the different key pairs [like we have done here](https://github.com/mattrglobal/node-bbs-signatures/blob/master/src/bls12381.ts#L60)

**Pull request closed without merging**